txstratum/client.py

```diff
@@ -0,0 +1,112 @@
+"""
```

Comment:
Shouldn't this be in `commons` too? We could later turn commons into a lib that merged mining can use too.

Reply:
I can move it there. Maybe the `commons/` folder should become a `hathor-commons` package itself.

Reply:
I haven't moved it yet because it is using structlog. How can we generalize the logging? Just use `import logging` instead?

Reply:
I think it's fine if it depends on structlog. At the very least we can think of replacing it when moving it out from the project.

Reply:
Ok. Done!